openrtb_ext/imp_cointraffic.go

```diff
@@ -0,0 +1,5 @@
+package openrtb_ext
+
+type ExtImpCointraffic struct {
+	PlacementId string `json:"PlacementId"`
```

Comment:
Please consider using all lower case "placementid" or snake case "placementId" for bidder parameters.